### ISSUE TITLE
Customized sorting CommitGroups not working?

### ISSUE THREAD

**Issue by maicol07** [git-chglog/git-chglog#140]
## Expected Behavior
Commit groups sorted as I have specified in the options

## Actual Behavior
Commit groups are not sorted

## Steps to Reproduce (including precondition)
1. Use the following options in your config file:
```yaml
commit_groups:
    sort_by: Custom
    title_order:
        - feat
        - update
        - perf
        - fix
        - refactor
        - build
        - chore
    title_maps:
        feat: ✨ Features
        fix: 🐛 Bug Fixes
        perf: ⚡ Performance Improvements
        refactor: ♻ Code Refactoring
        update: 🔄 Updates
        build: 👷 Building scripts changes
        chore: Other changes
```
2. Generate the changelog

## Your Environment
- OS: Windows 10
- git-chglog version: 0.14

**Comment by mavogel** [comment]
Can confirm this on OSX as well. Related to #69

**Comment by clok** [comment]
Is this a possible collision with the `sort` features we added recently? I haven't looked at the code yet, so I am just following a gut feeling. 

@khos2ow thoughts here?

**Comment by khos2ow** [comment]
I did a little bit of digging and it was not related to new `sort` but rather to accidentally removing the config [here](https://github.com/git-chglog/git-chglog/pull/52/files#diff-9fe5d92b3f46e081e851ff1b7e929068bef6da4b0c0ba3a2b75d900bf459f8beL274)

**Comment by clok** [comment]
> I did a little bit of digging and it was not related to new `sort` but rather accidentally removed the config [here](https://github.com/git-chglog/git-chglog/pull/52/files#diff-9fe5d92b3f46e081e851ff1b7e929068bef6da4b0c0ba3a2b75d900bf459f8beL274).

Nice find.

**Comment by clok** [comment]
Thank you for reporting this issue. We have included the fix for this within the [v0.14.2](https://github.com/git-chglog/git-chglog/releases/tag/v0.14.2) release.
